nav2_mppi_controller/src/noise_generator.cpp

```diff
@@ -68,9 +74,9 @@ void NoiseGenerator::setNoisedControls(
 {
   std::unique_lock<std::mutex> guard(noise_lock_);
 
-  xt::noalias(state.cvx) = control_sequence.vx + noises_vx_;
-  xt::noalias(state.cvy) = control_sequence.vy + noises_vy_;
-  xt::noalias(state.cwz) = control_sequence.wz + noises_wz_;
+  state.cvx = noises_vx_.rowwise() + control_sequence.vx.transpose();
```

Comment:
It is not sensible to store the noises in such a way that we don't need to rowwise for addition?

Reply:
noises are stored in the correct order. But in Eigen, if we directly add a vector in a matrix then it throws the EIGEN_ASSERTION errors due to mismatch in no. of cols and rows of operands. `.rowwise()` specifies to perform row-wise addition of a vector with matrix.